docs/current/742989-cookbook.md

```diff
@@ -625,6 +625,31 @@ Set the Hashicorp Vault URI, namespace, role and secret identifiers as host envi
 
 [Learn more](./guides/723462-use-secrets.md)
 
+### Mount directories as secrets in a container
+
+The following code listing demonstrates how to securely mount directories as secrets in a container. The directory structure will be exposed:
```

Comment:
`The directory structure will be exposed` - what does this mean? Do you mean that you can see the directory/file names, but not read the contents? Or something else?

Reply:
Exactly, the content will be scrubbed, not the file names neither the directory structure. Updated accordingly 🙏